app/models/tenant.rb

```diff
@@ -90,8 +90,16 @@ def all_subprojects
     self.class.descendants_of(self).where(:divisible => false)
   end
 
+  def regional_tenants
+    self.class.regional_tenants(self)
+  end
+
+  def self.regional_tenants(tenant)
+    where(arel_table.grouping(arel_attribute(:name).eq(tenant.name)))
```

Comment:
Let's make this a case insensitive match so that it's consistent with users and and groups

Reply:
done 👍